install

```diff
@@ -399,26 +399,107 @@ if [ ${version} -gt 4 ]; then
 
     echo "Configure ${nic} to use networkd ..."
     mkdir -p /etc/netplan
+    
+    # use preditable naming so doesn't break on reboot
+    nicMac=$(cat /sys/class/net/$nic/address | tr -d ':')
+    UUID=$(cat /proc/sys/kernel/random/uuid)
+
+    xmlstarlet ed -L \
```

Comment:
You can use OMV helper functions here, e.g. [omv_config_add_node_data](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/usr/share/openmediavault/scripts/helper-functions#L507) or [omv_config_add_node](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/usr/share/openmediavault/scripts/helper-functions#L440) and [omv_config_add_key](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/usr/share/openmediavault/scripts/helper-functions#L478).

Examples:
https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault-shairport/usr/share/openmediavault/confdb/create.d/conf.service.shairport.sh#L40
https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault-sharerootfs/debian/openmediavault-sharerootfs.postinst#L54